src/index.js

```diff
@@ -169,7 +169,7 @@ class Cropper extends React.Component {
   onWheel = e => {
     e.preventDefault()
     const point = Cropper.getMousePoint(e)
-    const newZoom = this.props.zoom - e.deltaY / 200
+    const newZoom = this.props.zoom - (e.deltaY * this.props.zoomSpeed) / 200
```

Comment:
I don't really like the hardcoded `200` value. Do you think we could combine it with the `zoomSpeed` property (and keep "approximatively" the same behavior as today)?

Reply:
I don't see, how we can combine this.
We could extract the hardcoded number to a class constant, but this is out of scope of this PR in my opinion.

Reply:
OK it's fine to me.